integration/client-cli/admin-cli/src/main/resources/META-INF/services/javax.ws.rs.ext.Providers

```diff
@@ -1,23 +0,0 @@
-org.jboss.resteasy.plugins.providers.jackson.ResteasyJackson2Provider
```

Comment:
Why is this no longer needed?

Reply:
The `keycloak-admin-cli` does not use JAX-RS. See comment https://github.com/keycloak/keycloak/pull/10917#issuecomment-1081904455.